examples/cluster/cluster.yaml

```diff
@@ -9,6 +9,7 @@ spec:
       cidrBlocks: ["${SERVICE_CIDR}"]
     pods:
       cidrBlocks: ["${CLUSTER_CIDR}"]
+    serviceDomain: ${DNS_DOMAIN}
```

Comment:
Hi @datianshi,

Can you please rename the environment variable to `SERVICE_DOMAIN`?